organizationinvitation/client.go

```diff
@@ -115,3 +115,42 @@ func (c *Client) Revoke(ctx context.Context, params *RevokeParams) (*clerk.Organ
 	err = c.Backend.Call(ctx, req, invitation)
 	return invitation, err
 }
+
+type ListFromInstanceParams struct {
+	clerk.APIParams
+	clerk.ListParams
+	Statuses *[]string `json:"statuses,omitempty"`
+	Query    *string   `json:"query,omitempty"`
+	OrderBy  *string   `json:"order_by,omitempty"`
+}
+
+func (p *ListFromInstanceParams) ToQuery() url.Values {
+	q := p.ListParams.ToQuery()
+
+	if p.Statuses != nil && len(*p.Statuses) > 0 {
+		q["status"] = *p.Statuses
+	}
+
+	if p.Query != nil {
+		q.Add("query", *p.Query)
+	}
+
+	if p.OrderBy != nil {
+		q.Add("order_by", *p.OrderBy)
+	}
+
+	return q
+}
+
+// ListAllFromInstance lists all the organization invitations from the current instance
+func (c *Client) ListFromInstance(ctx context.Context, params *ListFromInstanceParams) (*clerk.OrganizationInvitationList, error) {
```

Comment:
:question: Have we considered reusing the existing `organizationinvitation.List` method?

We could return all organization invitations for an instance if `ListParams.OrganizationID == ""`. Granted, the API would be better if the `OrganizationID` was a pointer to a string, but perhaps a single `List` method reduces the API surface?

What do you think?

Reply:
I think the issue here is that we have some additional paramaters that are not contained on the one with organizationID

Reply:
For now I think it's best to have both